firefox_ui_tests/functional/keyboard_shortcuts/test_browser_window.py

```diff
@@ -50,4 +50,4 @@ def has_input_selected(mn):
             """)
             return selection_name == "input"
 
-        self.wait_for_condition(has_input_selected)
+        Wait(self.marionette).until(has_input_selected)
```

Comment:
So this change looks fine, but have you run the tests locally yourself? At least this test will fail because `Wait` has not been imported.